common/Makefile.common.mk

```diff
@@ -71,7 +71,7 @@ code-gen:
 	@echo Updating the CRD files with the OpenAPI validations
 	operator-sdk generate crds
 	@echo Updating/Generating a ClusterServiceVersion YAML manifest for the operator
-	operator-sdk generate csv --csv-version ${CSV_VERSION} --update-crds
+	operator-sdk generate csv --csv-version ${CSV_VERSION} --update-crds --make-manifests=false
```

Comment:
`operator-sdk` v0.15.x don't support `--make-manifests=false`, I'll merge this PR, and then revert this arg